src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,14 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.14.4
+
+This is a bugfix release:
+
+* Fix stale IPtable rules along with globalIP leak which can sometimes happen as part of stress testing.
+* Handle out of order remote endpoint notifications properly in various Route Agent handlers.
+* Ensure that reverse path filtering setting is properly applied on the relevant network interfaces.
```

Comment:
Personally, I feel the original description captured in the release-notes is more readable.

CC: @dfarrell07 @tpantelis 

```
Submariner now ensures that reverse path filtering setting is properly applied on the `vx-submariner` and `vxlan-tunnel` interfaces after they are created. This fix was necessary for RHEL9 nodes where the setting was sometimes getting overwritten.
```